src/js/utils/computed-style.js

```diff
@@ -26,9 +26,9 @@ function computedStyle(el, prop) {
   }
 
   if (typeof window.getComputedStyle === 'function') {
-    const cs = window.getComputedStyle(el);
+    const computedStyleValue = window.getComputedStyle(el);
```

Comment:
maybe this should just be called `style`? That is what they use on `mdn`

Reply:
Are you referring to this page?
https://developer.mozilla.org/fr/docs/Web/API/Window/getComputedStyle

They also use `cs` in the examples, but I'm not a fan of very short abbreviations or too much generic term where you have to go back to the declaration to understand exactly the variable.

Do you want me to change my commit to use the name `style`?

Reply:
I think this is fine.